bins/packages/qsfs/qsfs.sh

```diff
@@ -0,0 +1,30 @@
+QSFS_VERSION="0.2.0-rc1"
```

Comment:
v0.3.0 was released https://github.com/threefoldtech/0-stor_v2/releases/tag/v0.3.0 for 0-stor does that reflect on qsfs version?

Reply:
well, that's the version used by quatnum-storage https://github.com/threefoldtech/quantum-storage/blob/qsfs-docker/docker/Dockerfile#L4

Reply:
Updated, i had to rebuild qsfs docker to use latest version of zstor